Jenkinsfile

```diff
@@ -2,9 +2,7 @@
 
 common {
   slackChannel = '#c3-alerts'
-  downStreamRepos = ["metadata-service", "kafka-rest",
-    "confluent-security-plugins", "ce-kafka-http-server", "secret-registry",
-    "confluent-cloud-plugins"]
+  downStreamRepos = ["metadata-service", "kafka-rest", "ce-kafka-http-server", "secret-registry"]
```

Comment:
Keeping kafka-rest in Jenkinsfile because schema-registry was removed in this file (schema-registry deprecated Jenkins so this repo is unable to start downstream builds in Jenkins for schema-registry).